ibft/event_loop.go

```diff
@@ -54,14 +54,20 @@ loop:
 
 		var wg sync.WaitGroup
 		if i.MsgQueue.MsgCount(msgqueue.IBFTMessageIndexKey(i.State.Lambda, i.State.SeqNumber, i.State.Round)) > 0 {
+			i.Logger.Debug("adding ibft message to event queue - waiting for done")
 			wg.Add(1)
-			i.eventQueue.Add(func() {
+			if added := i.eventQueue.Add(func() {
```

Comment:
@nivBlox I don't think we should return bool from add... what's the rational?

Reply:
we using wait group for those functions and in case the msg is not added (cause of queue stopped) we have blocked function